Support for module.exports functions

filmaj opened the pull request:
This fixes #10, fyi @romainl @rkulla

filmaj committed:
Support for module.exports functions

This fixes #10.

romainl commented:
Thank you @filmaj.

Could you add a test snippet to `index.js` and update the `README.md` to make this PR complete?

filmaj commented:
Sure thing, I think I added what was necessary. I'm not 100% sure if the `tags` file output is correct - I see two entries related to the test function name `mod_exports_func` that I added:

```
➜ make && grep mod_exp tags
[ 2019-04-22 - 08:44:39 ] 'tags' file updated.
exports	index.js	/^module.exports = function mod_exports_func() {$/;"	M
mod_exports_func	index.js	/^module.exports = function mod_exports_func() {$/;"	F
```

Hmm, I guess it is also picked up as a method name - do you think that is OK?
